src/core/image.js

```diff
@@ -563,6 +666,12 @@ var PDFImage = (function PDFImageClosure() {
               buffer[i] ^= 0xff;
             }
           }
+
+          // Added by RamSoft (11/4/2015)
```

Comment:
Please remove these kinds of comments and set this as the author of the commit instead.